test/lib/eTestLib.js

```diff
@@ -75,6 +77,9 @@ const contractNames = [
     'EulStakes',
     'EulDistributor',
 
+    // Swap Handlers
+    'UniswapV3SwapHandler',
```

Comment:
How about using a prefix as a namespace instead of suffix, so `SwapHandlerUniswapV3`? I'm thinking like we do with "IRMBlah".

Reply:
good point :+1: